data/human/ships.txt

```diff
@@ -3474,6 +3474,68 @@ ship "Surveillance Drone"
 
 
 
+ship "Valkyrie"
+	sprite "ship/pirate valkyrie"
+	thumbnail "thumbnail/pirate valkyrie"
+	attributes
+		category "Medium Warship"
+		"cost" 3900000
+		"shields" 4800
+		"hull" 2800
+		"required crew" 9
+		"bunks" 26
+		"mass" 157
+		"drag" 4.9
+		"heat dissipation" .63
+		"fuel capacity" 500
+		"cargo space" 41
+		"outfit space" 400
+		"weapon capacity" 170
+		"engine capacity" 76
+		weapon
+			"blast radius" 80
+			"shield damage" 800
+			"hull damage" 400
+			"hit force" 1200
+	outfits
+		"Quad Blaster Turret" 2
```

Comment:
```suggestion
		"Heavy Anti-Missile Turret" 2
```

Just needs a torpedo rack or two, and this is all set to be a heavy artillery that pounds enemies from afar with a constant stream of torpedoes while its fighters harass and pin them down. Get a good stream of torpedoes going, and any ship that gives its four fighters problems will probably be hurting long before they get close enough to cause the Valkyrie problems.